Dashlane/AccountMigration/VVM/MasterPasswordMigrationView.swift

```diff
@@ -58,7 +58,7 @@ extension MasterPasswordMigrationView: NavigationBarStyleProvider {
 struct MasterPasswordMigrationView_Previews: PreviewProvider {
     static var previews: some View {
         MasterPasswordMigrationView(title: "Create a Master Password for Dashlane",
-                                    subtitle: "Your account rights have changed. Create a strong Master Password to log in to Dashlane going forward.",
+                                    subtitle: "Your account rights have changed. Create a strong Master Password to log into Dashlane going forward.",
```

Comment:
There are differences of opinion about whether `log in` is a magical verb that should be treated as separate from `to`. I personally believe that `log` is a verb and you can `log into` a thing. If your style owner objects, I'm happy to drop this.